src/pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -118,27 +118,28 @@ def __init__(
         self._annealing_strategy = annealing_strategy
         self._avg_fn = avg_fn or self.avg_fn
         self._device = device
-        self._model_contains_batch_norm = None
-        self._average_model = None
+        self._max_epochs: int
+        self._model_contains_batch_norm: bool
+        self._average_model = pl.LightningModule()
```

Comment:
Instead, I think we could do:
```suggestion
        self._average_model: pl.LightningModule
```

Reply:
Maybe like this, just to be on the safe side?

```suggestion
        self._average_model: "pl.LightningModule"
```